server/backend/testspg/router/test_router_application.py

```diff
@@ -138,8 +156,8 @@ def test_get_fam_application_user_role_assignment_no_role_assignments(
     access_roles_fom_dev_only = ["FOM_DEV_ADMIN"]
 
     # test no user role assignment for the application
-    role_assignment_end_point = endPoint + \
-        f"/{TEST_FOM_DEV_APPLICATION_ID}/user_role_assignment"
+    role_assignment_end_point = end_point + \
+        f"/{FOM_DEV_APPLICATION_ID}/user_role_assignment"
```

Comment:
oh maybe this endpoint is the "get_application_role_assignment_end_point" ?

Reply:
Hm... I was only refactoring for variable name with inconsistent python snake_case at "endPoint" using VScode build-in automatic refactoring, and wasn't intended to touch other tests to increase the scope; there were already many files for refactoring this time.
But, sure, this can be changed as it located at the same file same as the new tests since I did create one convenient variable for it. I will need to tests them.